### DISCUSSION TITLE
Pipe stacktrace to file when running command in shell mode

### DISCUSSION THREAD

**Discussion by Ha5hBr0wn** [helix-editor/helix#7232] [General]
I sometimes use python file I have opened in helix as an ipython like shell and run it with !python playground.py.

It's nice to not have to switch out of the file to test the output of small example python code when developing. The problem is that when there is an error it doesn't insert the stack trace in the file but instead shows it in the status line and it gets truncated. How can I get the stack trace to show up in the file so I don't need to leave the file?

**Comment by kirawi** [comment]
You can write a script to map sderr to stdout and it should work.
